index.html

```diff
@@ -4,13 +4,48 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Instagram Clone</title>
-    <!-- don't forget to link your styles -->
+   <link rel="stylesheet" href="styles/style.css">
   </head>
-  <body>
-    Instagram Capture and Share the World's Moments Instagram is a fast, beautiful and fun way to share your life with
-    friends and family. Take a picture or video, choose a filter to transform its look and feel, then post to Instagram
-    &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-    Oh yeah, did we mention it's free? Download on the App Store Get it on Google Play About Us Support Blog Press API
-    Jobs Privacy Terms &copy; 2014 Instagram
-  </body>
+  <body><header>
+  <div class="nav-bar">
+      <nav>
+        <ul>
+          <li><img src="/images/brand.png" alt="Instagram-logo"class="logo"></li>
```

Comment:
Por temas de compatibilidad entre dispositivos. (windows vs mac vs linux) acostumbrate a usar una de las siguientes sintaxis para esto:

```suggestion
          <li><img src="./images/brand.png" alt="Instagram-logo"class="logo"></li>
```

O...
```suggestion
          <li><img src="images/brand.png" alt="Instagram-logo"class="logo"></li>
```

A mi por ejemplo tal cual lo tienes no me aparecen las imagenes :(